deploy/opencraft.yml

```diff
@@ -14,6 +14,10 @@
       UNATTENED_UPGRADES_ERRORS_RELAY_TO: "{{ inventory_hostname }}"
     - name: forward-server-mail
       FORWARD_MAIL_MYDOMAIN: "{{ inventory_hostname }}"
+      when: vagrant_mode is undefined or not vagrant_mode
     - name: geerlingguy.nginx
       nginx_remove_default_vhost: true
+      when: vagrant_mode is undefined or not vagrant_mode
     - opencraft
+    - name: pmbauer.tarsnap
+      when: vagrant_mode is not undefined or vagrant_mode
```

Comment:
We're running tarsnap for vagrant?

Reply:
I did this because of this comment https://github.com/open-craft/opencraft/blob/3666dd84d8d986167b8231b8f02f46ac332ec2aa/deploy/install_tarsnap.yml#L1-L4
I think I misinterpreted it. Reading it again it does seem like it should be possible to optionally install tarsnap on Vagrant.

Reply:
What I meant was, *should* we be? I don't think we had tarsnap running on vagrant dev environments before, and I don't think we even need it for Ocim itself because it's stateless AFAIK.

Reply:
@xitij2000 I think the confusion is that, I think you meant to put `when: vagrant_mode is undefined or not vagrant_mode` here.

Reply:
What I understood from that comment is that keeping the tarsnap Ansible script separate allows us to install tarsnap on Vagrant for testing, so we shouldn't restrict tarsnap based on whether the deployment is via Vagrant or not. 
If ``tarsnap_tarsnapper_conf`` is not configured, then, in any case, the tarsnap scripts and chron config won't be set up. If I add ``when: vagrant_mode is undefined or not vagrant_mode`` then it will be hard to set up tarsnap for testing inside vagrant (using this play at least).

Reply:
@xitij2000 Ah, I see what you mean. In that case :+1: